projects/bucket-protocol/index.js

```diff
@@ -181,6 +183,8 @@ function convertUnderlyingAssets(coin) {
   if(coin === SCALLOP_sbUSDT) return ADDRESSES.sui.suiUSDT
   // xBTC
   if(coin === xBTC_ADDRESS) return ADDRESSES.sui.WBTC
+  // mUSD
+  if(coin === mUSD_ADDRESS) return ADDRESSES.sui.USDC
```

Comment:
we have priice support for MUSD & xBTC, so this nis not needed:

https://coins.llama.fi/prices/current/sui:0xe44df51c0b21a27ab915fa1fe2ca610cd3eaa6d9666fe5e62b988bf7f0bd8722::musd::MUSD,sui:0x876a4b7bce8aeaef60464c11f4026903e9afacab79b9b142686158aa86560b50::xbtc::XBTC